openquake/commonlib/source.py

```diff
@@ -845,7 +884,7 @@ def init_serials(self):
         start = 0
         for sg in self.src_groups:
             for src in sg:
-                nr = src.num_ruptures
+                nr = src.count_ruptures()
```

Comment:
Please check if you can reuse `num_ruptures` instead

Reply:
Restored `num_ruptures`